mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -231,4 +241,57 @@ public void searchShouldNotReturnResultsFromOtherUsers() throws Exception {
         assertThat(metaDatas.get(0).getPath()).isEqualTo(MailboxPath.inbox(session));
     }
 
+    @ContractTest
+    public void updateAnnotationsShouldUpdateStoredAnnotation() throws MailboxException {
+        MailboxPath inbox = setUpMailbox();
```

Comment:
If we move to inline the code, it will have more than 2 duplicated block of code.
So what if I change that method to "setUpMailboxWithAnnotationSupport"? cause I still prefer to separate duplicated code to a function.